gammapy/datasets/load.py

```diff
@@ -119,6 +120,18 @@ def load_psf_fits_table():
     return fits.open(filename)
 
 
+def load_aeff2D_fits_table():
+    """Load an example aeff2D FITS file..
+
+    Returns
+    -------
+    hdu_list : `~astropy.io.fits.HDUList`
+        aeff2D file contents.
+    """
+    filename = get_path('irfs/aeff2D.fits')
```

Comment:
Yes, use `git add -f` to add this file and fix this test error on travis-ci:
https://travis-ci.org/gammapy/gammapy/jobs/61775130#L1033